docs/Dirb.md

```diff
@@ -0,0 +1,36 @@
+# How to install and use Dirb for Mac OS
+
+## Installation 🔧
+
+All you need to do is follow the commands below, and Dirb will be installed in no time!
+
+```sh
+curl -L https://sourceforge.net/projects/dirb/files/dirb/2.22/dirb222.tar.gz/download -o dirb222.tar.gz
+tar xvzf dirb222.tar.gz
+chmod 766 dirb222
+cd dirb222
+find . -type d -exec chmod 766 {} \;
+chmod +x configure
+./configure
+make && make install
+```
+
+---
+
+## How to use Dirb 👨‍💻
+
+For this course, we are providing you with a wordlist of common URL directories, which can be found [here][1], to be used with Dirb. Feel free to use your own though 😉 !
+
+To use Dirb, we need an URL and a wordlist. What the tool does now is try to access each and every URL directory present in the wordlist to see which ones are available. Be sure to only use Dirb with SecDevLabs machines and for educational purposes.
```

Comment:
Would you mind confirming it later? I think we may just need an URL to run `Dirb`. If none is passed as an argument it may use common.txt by default (not sure..). 

Additionally, how does this tool know that a URL is "available"? Status code? It would be nice if we are able to explain that to developers.

Reply:
Nice catch! I'll add some explanation!